*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to /tmp/code-caabcb63-a63f-4dd2-b0fa-3fe2a36acdbd
Extension host agent listening on /tmp/code-caabcb63-a63f-4dd2-b0fa-3fe2a36acdbd

[18:56:56] 




[18:56:56] Extension host agent started.
[18:56:56] [<unknown>][cb115a26][ManagementConnection] New connection established.
[18:56:57] [<unknown>][2bd136aa][ExtensionHostConnection] New connection established.
[18:56:57] [<unknown>][2bd136aa][ExtensionHostConnection] <1902> Launched Extension Host Process.
[18:58:14] [<unknown>][cb115a26][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[18:58:14] [<unknown>][2bd136aa][ExtensionHostConnection] <1902> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[18:58:14] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[18:58:14] Last EH closed, waiting before shutting down
[18:58:31] [<unknown>][0a9eda5b][ManagementConnection] New connection established.
[18:58:31] [<unknown>][c8d234f4][ExtensionHostConnection] New connection established.
[18:58:31] [<unknown>][c8d234f4][ExtensionHostConnection] <2094> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty 4 on pty host
stack trace: CodeExpectedError: Could not find pty 4 on pty host
    at I.W (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateTitle (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[18:58:33] Error [CodeExpectedError]: Could not find pty 4 on pty host
    at I.W (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateTitle (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[18:59:25] [<unknown>][0a9eda5b][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[18:59:25] [<unknown>][c8d234f4][ExtensionHostConnection] <2094> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[18:59:25] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[18:59:25] Last EH closed, waiting before shutting down
[18:59:38] [<unknown>][b5e0e484][ManagementConnection] New connection established.
[18:59:38] [<unknown>][90fcb52a][ExtensionHostConnection] New connection established.
[18:59:38] [<unknown>][90fcb52a][ExtensionHostConnection] <2299> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty 5 on pty host
stack trace: CodeExpectedError: Could not find pty 5 on pty host
    at I.W (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateTitle (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[18:59:40] Error [CodeExpectedError]: Could not find pty 5 on pty host
    at I.W (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12275)
    at I.updateTitle (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at el.s (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80918)
    at el.q (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80441)
    at ds.value (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79843)
    at x.B (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at x.fire (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.C (file:///home/ubuntu_user/.vscode-server/cli/servers/Stable-ddc367ed5c8936efe395cffeec279b04ffd7db78/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29877)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[18:59:45] [<unknown>][b5e0e484][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[18:59:45] [<unknown>][90fcb52a][ExtensionHostConnection] <2299> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[18:59:45] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[18:59:45] Last EH closed, waiting before shutting down
